src/framework/Framework.Models/ValidationExpressionsValidator.cs

```diff
@@ -0,0 +1,30 @@
+/********************************************************************************
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+using System.Text.RegularExpressions;
+
+namespace Org.Eclipse.TractusX.Portal.Backend.Framework.Models;
+
+public static class ValidationExpressionsValidator
+{
+    public static bool IsValidCompanyName(string companyName)
```

Comment:
I suggest to define this as extension-method:
```suggestion
    public static bool IsValidCompanyName(this string companyName)
```
this way it can be used like so:
```
using Org.Eclipse.TractusX.Portal.Backend.Framework.Models;

var companyName = "Acme Corp";
if (!companyName.IsValidCompanyName())
{
}